Can not specify the output file as the files in the repo

**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
Want to set he output file as the `README.md` in the repo, but there is error:

```
[2018-03-18 17:01:04] [ERROR] [worker.go:105] API error (500): Could not find the file README.md in container f77919d162bca03985e06cb6a5370313f916293c7920e85a926079b288b04e0b
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
/assign @supereagle
/priority p0

--- a/pkg/worker/stage/stage.go
+++ b/pkg/worker/stage/stage.go
@@ -249,8 +249,16 @@ func (sm *stageManager) ExecPackage(builderImage *api.BuilderImage, buildInfo *a
 
 	// Copy the build outputs if necessary.
 	// Only need to copy the outputs not in the current workspace. The outputs must be absolute path of files or folders.
-	for _, output := range packageStage.Outputs {
-		if err = runner.CopyFromContainer(sm.dockerManager.Client, cid, output, cloneDir+"/"); err != nil {
+	for _, ot := range packageStage.Outputs {
+		cloneDir := cloneDir + "/"
+		if strings.HasPrefix(ot, "./") {
+			ot = strings.TrimPrefix(ot, "./")
+			ot = cloneDir + ot
+		} else if !strings.HasPrefix(ot, "/") {
+			ot = cloneDir + ot
+		}
+
+		if err = runner.CopyFromContainer(sm.dockerManager.Client, cid, ot, cloneDir); err != nil {
 			return err
 		}
 	}

